### PR TITLE
Add build package.json to blacklist

### LINKED ISSUE #297: Ambiguous resolution
image

### PR DESCRIPTION
**Pull request by tbergq**
closes #297

### CODE REVIEW

**Comment by mrtnzlml** [comment]
@tbergq Closes #297?

File: rn-cli.config.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const sharedBlacklist = [
   /node_modules[/\\]react[/\\]dist[/\\].*/,
   /website\/node_modules\/.*/,
   /heapCapture\/bundle\.js/,
+  /\.build[/\\]package.json/,
```
Comment (mrtnzlml) [inline rn-cli.config.js]: Should we ignore the `.build` completely?

Comment (tbergq) [reply rn-cli.config.js]: No, we need to keep the package.json in git, so we can increment version number

Comment (mrtnzlml) [reply rn-cli.config.js]: It's not `.gitignore`. I mean - should we ignore it completely in the `rn-cli`?

Comment (tbergq) [reply rn-cli.config.js]: Right, now I understood. Yes, we might as well do that.

**Comment by tbergq** [comment]
Yes, closes #297, it is getting too late for me 😖

**Pull request merged**